go.mod

```diff
@@ -3,53 +3,70 @@ module github.com/kubeflow/kfserving
-	golang.org/x/net v0.0.0-20200520004742-59133d7f0dd7
-	golang.org/x/time v0.0.0-20191023065245-6d3f0bb11be5 // indirect
-	google.golang.org/grpc v1.27.0
+	github.com/spf13/cobra v1.0.0
+	github.com/stretchr/testify v1.5.1
+	golang.org/x/net v0.0.0-20200904194848-62affa334b73
+	google.golang.org/grpc v1.31.1
 	google.golang.org/protobuf v1.25.0
-	istio.io/api v0.0.0-20191115173247-e1a1952e5b81
-	istio.io/client-go v0.0.0-20191120150049-26c62a04cdbc
+	istio.io/api v0.0.0-20200715212100-dbf5277541ef
+	istio.io/client-go v0.0.0-20201005161859-d8818315d678
 	istio.io/gogo-genproto v0.0.0-20191029161641-f7d19ec0141d // indirect
-	k8s.io/api v0.17.2
-	k8s.io/apiextensions-apiserver v0.0.0-20190918161926-8f644eb6e783
-	k8s.io/apimachinery v0.17.2
-	k8s.io/client-go v0.17.2
-	k8s.io/code-generator v0.18.6 // indirect
+	k8s.io/api v0.18.8
+	k8s.io/apiextensions-apiserver v0.18.8
+	k8s.io/apimachinery v0.18.8
+	k8s.io/client-go v11.0.1-0.20190805182717-6502b5e7b1b5+incompatible
 	k8s.io/klog v1.0.0
-	k8s.io/kube-openapi v0.0.0-20200410145947-61e04a5be9a6
-	knative.dev/pkg v0.0.0-20191217184203-cf220a867b3d
-	knative.dev/serving v0.11.0
-	sigs.k8s.io/controller-runtime v0.4.0
+	k8s.io/kube-openapi v0.0.0-20200410145947-bcb3869e6f29
+	knative.dev/pkg v0.0.0-20200922164940-4bf40ad82aab
+	knative.dev/serving v0.18.0
+	sigs.k8s.io/controller-runtime v0.6.3
+)
+
+replace (
+	k8s.io/api => k8s.io/api v0.18.8
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.18.8
+	k8s.io/apimachinery => k8s.io/apimachinery v0.18.8
+	k8s.io/apiserver => k8s.io/apiserver v0.18.8
+	k8s.io/cli-runtime => k8s.io/cli-runtime v0.18.8
+	k8s.io/client-go => k8s.io/client-go v0.18.8
+	k8s.io/cloud-provider => k8s.io/cloud-provider v0.18.8
+	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.18.8
+	k8s.io/code-generator => k8s.io/code-generator v0.18.8
+	k8s.io/component-base => k8s.io/component-base v0.18.8
+	k8s.io/cri-api => k8s.io/cri-api v0.18.8
+	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.18.8
+	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.18.8
+	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.18.8
+	k8s.io/kube-proxy => k8s.io/kube-proxy v0.18.8
+	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.18.8
+	k8s.io/kubectl => k8s.io/kubectl v0.18.8
+	k8s.io/kubelet => k8s.io/kubelet v0.18.8
+	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.18.8
+	k8s.io/metrics => k8s.io/metrics v0.18.8
+	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.18.8
+	k8s.io/utils => k8s.io/utils v0.0.0-20200324210504-a9aa75ae1b89
+	k8s.io/test-infra => k8s.io/test-infra v0.0.0-20200803112140-d8aa4e063646
+	k8s.io/klog => k8s.io/klog v1.0.0
+	// based on https://github.com/openshift/cluster-ingress-operator/pull/429/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R34
+	sigs.k8s.io/controller-runtime => github.com/zchee/sigs.k8s-controller-runtime v0.6.1-0.20200623114430-46812d3a0a50
```

Comment:
why do we use the controller-runtime fork ?

Reply:
That’s due to https://github.com/kubernetes-sigs/controller-runtime/issues/1033

Reply:
Is the fork still there? Seems not work for me:
```
➜  kfserving git:(migrate-crd-v1) go mod verify
go: github.com/zchee/sigs.k8s-controller-runtime@v0.6.1-0.20200623114430-46812d3a0a50: reading https://goproxy.io/github.com/zchee/sigs.k8s-controller-runtime/@v/v0.6.1-0.20200623114430-46812d3a0a50.mod: 404 Not Found
	server response: not found: unknown revision 46812d3a0a50
```

Reply:
Hmm, `go mod verify` works for me and the fork seems to be there - https://github.com/openshift/cluster-ingress-operator/pull/429/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R34

Also the makefile commands like `make manager` work. I'm on go1.14.4. Maybe it's a go version thing?

Reply:
Thanks. I am on go1.14.7 and not sure why. But checked https://github.com/zchee/sigs.k8s-controller-runtime/tags and not found v0.6.1. Maybe because it's already under your go mod cache...